qulice-pmd/src/test/java/com/qulice/pmd/FilesTest.java

```diff
@@ -1,66 +0,0 @@
-/**
- * Copyright (c) 2011-2016, Qulice.com
- * All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions
- * are met: 1) Redistributions of source code must retain the above
- * copyright notice, this list of conditions and the following
- * disclaimer. 2) Redistributions in binary form must reproduce the above
- * copyright notice, this list of conditions and the following
- * disclaimer in the documentation and/or other materials provided
- * with the distribution. 3) Neither the name of the Qulice.com nor
- * the names of its contributors may be used to endorse or promote
- * products derived from this software without specific prior written
- * permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
- * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
- * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
- * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
- * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
- * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
- * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
- * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
- * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
- * OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-package com.qulice.pmd;
-
-import com.qulice.spi.Environment;
-import java.io.IOException;
-import java.util.Collection;
-import net.sourceforge.pmd.util.datasource.DataSource;
-import org.apache.commons.io.IOUtils;
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.Matchers;
-import org.junit.Test;
-
-/**
- * Test case for {@link Files} class.
- * @author Krzysztof Krason (Krzysztof.Krason@gmail.com)
- * @version $Id$
- * @since 0.7
- */
-public final class FilesTest {
-
-    /**
-     * Should provide only java files.
-     * @throws IOException In case of problem.
-     */
-    @Test
-    public void providesOnlyJavaFiles() throws IOException {
-        final String source = "class Cls{}";
-        final Environment env = new Environment.Mock()
-            .withFile("src/main/java/Main.java", source)
-            .withFile("src/main/resources/test.properties", "prop=1");
```

Comment:
@krzyk that's good, PMD has some useful non-java check, but we're not using them